src/storage/diffusion.md

```diff
@@ -0,0 +1,26 @@
+# Chain diffusion
```

Comment:
Would pipelining be linked or mentioned here (in the future)?

Reply:
How so? In the mdBook I think it is perfectly clear that it _is_ a part of chain diffusion section 

![Uploading image.png…]()

Reply:
I meant explained in 2.3.1

Reply:
Not sure if my confusion is related to yours here: Why is chain diffusion something related to "storage"? I do understand it explains how the mini-protocols are served by the consensus layer, so there is definitely a connection/overlap to the actual network protocol, but I don't see how that classifies as storage?

Reply:
I added another paragraph that I think makes it more clear:

> In a sense, the Storage layer has the data to provide the meaning of the
messages in the mini-protocol whereas the networking layer describes the kinds
of messages a protocol is composed by. It is possible to run an interaction of
the protocol exchanging data that does not follow the intended semantics (for
example an evil node sending all the chains it knows about instead of only the
best selection). That's why we describe here when each message should be emitted
and what information it should carry.